.github/workflows/electron-releases.yml

```
@@ -0,0 +1,41 @@
+name: Auto-update Releases JSON file
+on:
+  schedule:
+    - cron: '0 0 * * *'
+jobs:
+  autoupdate:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v1
+      with:
+        node-version: '12.x'
+    - name: Get npm cache directory
+      id: npm-cache
+      run: |
+        echo "::set-output name=dir::$(npm config get cache)"
+    - uses: actions/cache@v1
+      with:
+        path: ${{ steps.npm-cache.outputs.dir }}
+        key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+        restore-keys: |
+          ${{ runner.os }}-node-
+    - run: npm ci
+    - name: Update Releases JSON
+      run: npm run electron-releases
+    - name: Commit Changes to Releases JSON
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: |
+        echo "machine github.com login $GITHUB_ACTOR password $GITHUB_TOKEN" > ~/.netrc
+        chmod 600 ~/.netrc
+        git add static/releases.json
+        if test -n "$(git status -s)"; then
+          git config user.name "$GITHUB_ACTOR"
+          git config user.email "electron-bot@users.noreply.github.com"
+          git diff --cached
+          git commit -m "Update Electron Releases"
```

nit: If this commit is going directly onto the `master` branch, do we want to keep semantic commits?

---

Yes, you're right. It should be
```suggestion
          git commit -m "build: update Electron Releases"
```